pydm/widgets/slider.py

```diff
@@ -73,14 +80,157 @@ def __init__(self, parent=None, init_channel=None):
         self.setup_widgets_for_orientation(self._orientation)
         self.reset_slider_limits()
 
+        self.widget = None
+        self.input = None
+        self.label = None
+        self.button = None
```

Comment:
Would also help to give more descriptive names to these as well so it is easier to know what their purpose is.